### PR TITLE
Updated stolen supplies

### PR DESCRIPTION
**Pull request by Ketrai**
Removes the majority of phazon pieces. (getting the full set has an extremely tiny number of succeeding, and you can't get the circuit board anyways. I kept one so people know it still exists.) Adds high tier contraband you can find in trash piles to give an alternative way to obtain these. Hopefully encourages more people to mine to get points and resources for science. If you feel this isn't balanced enough, can always increase the probability of getting cosmetic or lesser items to dilute the pool.

### CODE REVIEW

**Comment by OrbisAnima** [comment]
+1

File: code/game/objects/items/contraband_vr.dm [hunk 1]
```diff
@@ -36,12 +36,12 @@
 						/obj/item/device/pda/mime,
 						/obj/item/device/pda/syndicate,
 						/obj/item/mecha_parts/chassis/phazon,
-						/obj/item/mecha_parts/part/phazon_head,
-						/obj/item/mecha_parts/part/phazon_left_arm,
-						/obj/item/mecha_parts/part/phazon_left_leg,
-						/obj/item/mecha_parts/part/phazon_right_arm,
-						/obj/item/mecha_parts/part/phazon_right_leg,
-						/obj/item/mecha_parts/part/phazon_torso,
```
Comment (SpadesNeil) [inline code/game/objects/items/contraband_vr.dm]: I don't want phazon parts removed. The way the phazon used to work is you just put the parts together and it was done, which was weird, but worked. If it has circuits now, add them. Don't remove the phazon.

Comment (Ketrai) [inline code/game/objects/items/contraband_vr.dm]: hm, although spades. the actual chance to get all the phazon pieces, even with something as much as 40 crates is so low that getting the whole set is like winning the lottery, exponential chance that is. I could math and put it all out for you, but the numbers wouldn't be very encouraging.

Comment (OrbisAnima) [inline code/game/objects/items/contraband_vr.dm]: wait, really, if you put it all you can make one? awesome.

Comment (SpadesNeil) [inline code/game/objects/items/contraband_vr.dm]: I know the odds are **extremely** slim. That's the point. Not all of the items you get from the stolen goods crate are useful.

Though phazon parts are still worth juicy research points.

Comment (Ketrai) [inline code/game/objects/items/contraband_vr.dm]: Alright, I'll fix the pull.

**Pull request merged**